README.md

```diff
@@ -1,3 +1,53 @@
-# Med API
+<section style="text-align: center;">
 
-TBD
+# Federation of State Medical Boards</br>MED API Documentation
+</section>
+
+This is the source for technical information for the Federation of State Medical Boards' Federation Credentials Verification Service (MED) RESTful API. 
```

Comment:
Did this come from the docs because it isn't related to FCVS directly. Only one of the resources.

"RESTful" this kills me. I don't care if it is in the docs that way we say "REST API". Provide a link if they don't know - https://en.wikipedia.org/wiki/Representational_state_transfer